functions/ts/build/helm_template.Dockerfile

```diff
@@ -19,6 +19,8 @@ WORKDIR /home/node/app
 
 # Install dependencies and cache them.
 COPY --chown=node:node package*.json ./
+# Make rw package work
```

Comment:
These work fine right now. Why is this change necessary?

Reply:
I needed rw.writeFile to put yamls into /tmp/ directory (sops works with files).
I looked up this function in [kpt-functions-sdk](https://github.com/GoogleContainerTools/kpt-functions-sdk).
It didn't work until I modified [package.json](https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/101/files#diff-6e89b8d725b5789ac81ff297fac241c1) and [package-lock.json](https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/101/files#diff-0613cd7774d07870a90667aa23032291) by adding `file:@types/rw`.  But since these files are shared across all ts-functions, it's necessary to have `@types/rw` in all dockerfiles, because it's referenced. I wasn't able to find a better solution. Maybe separate package.json for each ts-function? but I'm not sure where to put it.

Reply:
Filed https://github.com/GoogleContainerTools/kpt/issues/1026

To unblock you, you can add TODO in Dockerfiles linking to that issue.

Reply:
done